plotter/main.go

```diff
@@ -22,74 +24,47 @@ import (
 )
 
 var examples = []struct {
-	name   string
-	mkplot func() *plot.Plot
+	name string
+	p    *plot.Plot
 }{
-	{"example_logo", Example_logo},
-	{"example_functions", Example_functions},
-	{"example_boxPlots", Example_boxPlots},
-	{"example_groupedBoxPlots", Example_groupedBoxPlots},
-	{"example_groupedHorizontalBoxPlots", Example_groupedHorizontalBoxPlots},
-	{"example_quartPlots", Example_quartPlots},
-	{"example_groupedQuartPlots", Example_groupedQuartPlots},
-	{"example_groupedHorizontalQuartPlots", Example_groupedHorizontalQuartPlots},
-	{"example_verticalBoxPlots", Example_verticalBoxPlots},
-	{"example_verticalQuartPlots", Example_verticalQuartPlots},
-	{"example_horizontalBoxPlots", Example_horizontalBoxPlots},
-	{"example_horizontalQuartPlots", Example_horizontalQuartPlots},
-	{"example_points", Example_points},
-	{"example_errBars", Example_errBars},
-	{"example_bubbles", Example_bubbles},
-	{"example_histogram", Example_histogram},
-	{"example_barChart", Example_barChart},
-	{"example_stackedBarChart", Example_stackedBarChart},
-	{"example_heatMap", Example_heatMap},
+	{"example_logo", Example_logo()},
+	{"example_functions", Example_functions()},
+	{"example_boxPlots", Example_boxPlots()},
+	{"example_groupedBoxPlots", Example_groupedBoxPlots()},
+	{"example_groupedHorizontalBoxPlots", Example_groupedHorizontalBoxPlots()},
+	{"example_quartPlots", Example_quartPlots()},
+	{"example_groupedQuartPlots", Example_groupedQuartPlots()},
+	{"example_groupedHorizontalQuartPlots", Example_groupedHorizontalQuartPlots()},
+	{"example_verticalBoxPlots", Example_verticalBoxPlots()},
+	{"example_verticalQuartPlots", Example_verticalQuartPlots()},
+	{"example_horizontalBoxPlots", Example_horizontalBoxPlots()},
+	{"example_horizontalQuartPlots", Example_horizontalQuartPlots()},
+	{"example_points", Example_points()},
+	{"example_errBars", Example_errBars()},
+	{"example_bubbles", Example_bubbles()},
+	{"example_histogram", Example_histogram()},
+	{"example_barChart", Example_barChart()},
+	{"example_stackedBarChart", Example_stackedBarChart()},
+	{"example_heatMap", Example_heatMap()},
+}
+
+var formats = []string{
```

Comment:
It may or may not be simpler in the runtime depending on the degree of optimisation - if it's an array and the compiler doesn't notice that its a literal, or doesn't use that information then the array will be copied prior to executing the for loop.